.github/workflows/ci-sharkfuser.yml

```diff
@@ -90,17 +91,27 @@ jobs:
       working-directory: ${{ env.SHARKFUSER_DIR }}
       run: |
         sudo apt update
-        sudo apt install cmake ninja-build catch2 ${{matrix.additional-packages}}
+        sudo apt install cmake ninja-build catch2 wget ${{matrix.additional-packages}}
         python -m venv --prompt fusili .venv
         source .venv/bin/activate
         python -m pip install --upgrade pip
         pip install -r ./test_requirements.txt
 
+    - name: Install prebuilt iree
+      working-directory: ${{ env.SHARKFUSER_DIR }}
+      run: |
+        wget --no-verbose -O iree-dist.tar.xz $IREE_RELEASE_URL
+        mkdir iree-dist
+        tar -xf iree-dist.tar.xz -C iree-dist
+        rm iree-dist.tar.xz
+        echo "$SHARKFUSER_DIR/iree-dist/bin" >> $GITHUB_PATH
+
     - name: Build sharkfuser
       working-directory: ${{ env.SHARKFUSER_DIR }}
       run: |
         source .venv/bin/activate
         cmake -GNinja -S. -Bbuild \
+          -DIREE_DIR=$SHARKFUSER_DIR/iree-dist/lib/cmake \
```

Comment:
Is this really needed? I see an `INSTALL` target, can we use that to have it installed (system-wide) and used in Fusili through `find_package` / `find_program`? 

https://sourcegraph.com/github.com/iree-org/iree/-/blob/build_tools/cmake/CMakeLists.txt?L28

Reply:
If I'm not mistaken we'll need `-DIREE_DIR` here unless we move the downloaded package into system locations where `find_package` would look by default.  IIUC that `install` command will decide what's in the bundle we're downloading, I don't think we have a cmake install command available in what we get. But, if we add
```sh
sudo mv iree-dist/* /usr/local/
```
to the install block `find_package` should work without this `-D` variable (I think [CMAKE_SYSTEM_PREFIX_PATH docs](https://cmake.org/cmake/help/latest/variable/CMAKE_SYSTEM_PREFIX_PATH.html). Would you prefer that?